tests/pkcli/build_data/1.in/db/000.yml

```diff
@@ -446,6 +446,9 @@ host:
         users:
           devtech3: 3100
           fullstackdude: 3101
+          user_w_custom_image:
```

Comment:
just call the user `custom_image`. Prefixing with user is unnecessary and perhaps confusing.

Reply:
d20cddb